src/pool/srv_pool_map.c

```diff
@@ -229,16 +234,17 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 		case PO_COMP_ST_DOWNOUT:
 		case PO_COMP_ST_NEW:
 			/* Nothing to do, already UPIN */
-			D_INFO("Skip ADD_IN UPIN "DF_TARGET"\n",
+			D_INFO(DF_MAP ": Skip ADD_IN UPIN " DF_TARGET "\n", DP_MAP(pool_uuid, map),
```

Comment:
```suggestion
			D_INFO(DF_MAP ": Skip FINISH_REBUILD DF_TARGET "\n", DP_MAP(pool_uuid, map),
```

Reply:
Changed to reflect the `case`s better. Please double check.